Python/dtoa.c

```diff
@@ -330,7 +330,6 @@ Bigint {
 
 typedef struct Bigint Bigint;
 
-#ifndef Py_USING_MEMORY_DEBUGGER
```

Comment:
@mdickinson: Should we add a new macro to enable or disable the freelist, for debug purposes?